src/components/Picklist/index.js

```diff
@@ -446,12 +444,16 @@ class Picklist extends Component {
                             <MenuArrowButton
                                 arrow="up"
                                 onMouseEnter={this.handleScrollUpArrowHover}
-                                onMouseLeave={this.handleStopArrowScoll}
+                                onMouseLeave={() => {
+                                    this.stopArrowScoll();
+                                }}
```

Comment:
here a new function is recreated in each rerender, bind stopArrowScoll in constructor and use:
```onMouseLeave={this.stopArrowScoll}```